src/app/pages/LendingPage/assets/index.scss

```diff
@@ -0,0 +1,331 @@
+@import '../../../../styles/sass/custom-variables';
```

Comment:
Why do you use `scss`? We should use `Tailwind` as much as possible (and I see several definitions from this file where it's possible) or `styled components` for the rest.